spec/jobs/anonymized_data_download_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AnonymizedDataDownloadJob, type: :job do
```

Comment:
We should add specs here or delete this file

Reply:
@timcheadle Yes, I plan to fill out the generated specs (as well as add more custom ones). I am opening the PR up for review now to elicit early feedback whilst adding specs and testing in parallel.